### PR TITLE
Use theme gridColor and ticksColor in long time graph

### PR DESCRIPTION
**Pull request by yubiuser**
- [x] I have read and understood the [contributors guide](https://github.com/pi-hole/AdminLTE/blob/master/CONTRIBUTING.md), as well as this entire template.
- [x] I have made only one major change in my proposed changes.
- [x] I have commented my proposed changes within the code.
- [x] I have tested my proposed changes.
- [x] I am willing to help maintain this change if there are issues with it later.
- [x] I give this submission freely and claim no ownership.
- [x] It is compatible with the [EUPL 1.2 license](https://opensource.org/licenses/EUPL-1.1)
- [x] I have squashed any insignificant commits. ([`git rebase`](http://gitready.com/advanced/2009/02/10/squashing-commits-with-rebase.html))
- [x] I have Signed Off all commits. ([`git commit --signoff`](https://git-scm.com/docs/git-commit#Documentation/git-commit.txt---signoff))

---

**What does this PR aim to accomplish?:**
Use the gridColor and ticksColor defined in the theme css files for the long term data graph. The properties are already used for the graph on the dashboard (last 24h graph). Using the same code here to have the same style.

### CODE REVIEW

File: scripts/pi-hole/js/db_graph.js [hunk 1]
```diff
@@ -15,12 +15,6 @@ var interval = 0;
 
 var dateformat = "MMMM Do YYYY, HH:mm";
 
-// get the database min timestamp
```
Comment (rdwebdesign) [inline scripts/pi-hole/js/db_graph.js]: Why this was removed?

Are you reverting #1969 and #1976 ?

File: scripts/pi-hole/js/db_graph.js [hunk 2]
```diff
@@ -44,7 +38,7 @@ $(function () {
           moment().subtract(1, "month").endOf("month"),
         ],
         "This Year": [moment().startOf("year"), moment()],
-        "All Time": [moment(mintimestamp), moment()],
```
Comment (rdwebdesign) [inline scripts/pi-hole/js/db_graph.js]: @yubiuser 
Why this was removed?

Are you reverting #1969 and #1976 ?

File: scripts/pi-hole/js/db_graph.js (hunk 1 above)
Comment (yubiuser) [reply scripts/pi-hole/js/db_graph.js]: Git accident. Multiple git directories with different state of `devel`. Going to fix this..

File: scripts/pi-hole/js/db_graph.js (hunk 2 above)
Comment (yubiuser) [reply scripts/pi-hole/js/db_graph.js]: Fixed.

**Comment by yubiuser** [comment]
Thanks to @rdwebdesign who pointed out an other small glitch and provided a fix for this. The first line in the graphs is darker than the rest (zeroLine). This is fixed now as well

diff_dashboard_longtermgraph
.

**Review by DL6ER — APPROVED** [review]
## Before

Screenshot from 2021-11-21 07-12-39
Screenshot from 2021-11-21 07-14-13

## After

Screenshot from 2021-11-21 07-12-19
Screenshot from 2021-11-21 07-14-31

**Pull request merged**